share/sysbuild/CMakeLists.txt

```diff
@@ -24,8 +24,13 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE} COMPONENTS ${zephyr_modules
 
 project(sysbuild LANGUAGES)
 
-# Global list of images enabled in this multi image build system.
+# Global list of images enabled in this multi image build system, with internal variables used
+# for holding a list of added images and directories for processing includes.
 set(IMAGES)
+set(ITERATE_IMAGES)
+set_property(GLOBAL PROPERTY ITERATE_IMAGES ${ITERATE_IMAGES})
+set(ITERATE_IMAGE_DIRS)
+set_property(GLOBAL PROPERTY ITERATE_IMAGE_DIRS ${ITERATE_IMAGE_DIRS})
```

Comment:
I believe those lines can now be removed.
```suggestion
```

Reply:
Fixed